### PR TITLE
94 - Use ckanext fjelltopp theme

### LINKED ISSUE #24: Use react resource uploader

### PR DESCRIPTION
**Pull request by A-Souhei**
## Description

ckanext-who-romania adjusted for ckanext-fjelltopp-theme.

image
image
image

Relates https://github.com/fjelltopp/fjelltopp-infrastructure/pull/292
Closes https://github.com/fjelltopp/who-romania-ckan/issues/94
Closes https://github.com/fjelltopp/ckanext-who-romania/issues/24

## Checklist

Put an `x` in the boxes that apply to this pull request (you can also fill these out after opening the pull request).
You may not need to check all boxes.

- [x] The Jira ticket for this issue has been updated to "Ready to Review" or equivalent.
- [x] I have developed these changes in discussion with the appropriate project manager.
- [ ] My code follows the general Fjelltopp documentation (see Confluence).
- [ ] I have made corresponding changes to the Fjelltopp documentation (see Confluence).
- [ ] I have rebased this branch with master.
- [ ] New dependency changes have been committed.
- [ ] I have added automated tests that prove my fix is effective or that my feature works.
- [ ] New and existing tests pass locally with my changes.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own code.
- [x] I have assigned at least one reviewer.
- [x] I have assigned at least one label to this PR: "patch", "minor", "major".

### CODE REVIEW

**Comment by A-Souhei** [comment]
@mixmixmix the theme is enabled with this branch, still many fixes but when you have time , can you verify the palette if it is fine or requires changes?

**Comment by A-Souhei** [comment]
@mixmixmix , this is done , for your review when you have time.

File: ckanext/who_romania/assets/webassets.yml [hunk 1]
```diff
@@ -2,9 +2,13 @@ who-romania-theme:
   filter: cssrewrite
   output: who-romania/%(version)s-who-romania.css
   contents:
-    - css/inter.css
     - css/who-romania.css
 
+wrc-theme-palette:
+  output: zarr/%(version)s-zarr-palette.css
```
Comment (mixmixmix) [inline ckanext/who_romania/assets/webassets.yml]: this doesn't look right `zarr`

Comment (A-Souhei) [reply ckanext/who_romania/assets/webassets.yml]: I will fix that. Thanks!

**Review by mixmixmix — APPROVED** [review]
LGTM

**Pull request merged**